src/Blocks/BlockTripwireHook.h

```diff
@@ -83,14 +83,20 @@ class cBlockTripwireHookHandler :
 
 	virtual bool CanBeAt(cChunkInterface & a_ChunkInterface, const Vector3i a_RelPos, const cChunk & a_Chunk) override
 	{
-		auto Meta = a_Chunk.GetMeta(a_RelPos);
-		auto NeighborPos = AddFaceDirection(a_RelPos, MetadataToDirection(Meta), true);
+		NIBBLETYPE Meta = a_Chunk.GetMeta(a_RelPos);
+		Vector3i NeighborPos = AddFaceDirection(a_RelPos, MetadataToDirection(Meta), true);
+
 		if (!cChunkDef::IsValidHeight(NeighborPos.y))
 		{
 			return false;
 		}
+
 		BLOCKTYPE NeighborBlockType;
-		a_Chunk.UnboundedRelGetBlockType(a_RelPos, NeighborBlockType);
```

Comment:
On fail this would return false (0) since it's not halting here NeighbotBlockType could be 0 (which is air block type), and this would cause the return to succeed, potentially causing a bug of placing tripwire hooks on air.